app/services/locations/importer.rb

```diff
@@ -26,8 +38,10 @@ def call
         end
         location.save
       end
+    end
 
-      active_agency_ids = items.map { |item| item[:nomis_agency_id] }
+    def disable_unused_locations
+      active_agency_ids = locations.map { |item| item[:nomis_agency_id] }
```

Comment:
may be simplify to
active_agency_ids =  locations.map(&:nomis_agency_id)

Reply:
I don't think that is the same thing - it equates to calling `item.nomis_agency_id` rather than `item[:nomis_agency_id]`. And given `item` is just a hash we need to call the `[]` method passing the symbol argument.

Reply:
oh yes, you are quite right, my bad 👍